components/PoisList/PoisList.vue

```diff
@@ -92,6 +92,9 @@ export default defineNuxtComponent({
 
   methods: {
     onMenuChange(newCategoryId: number) {
+      if (!newCategoryId)
+        return
+
```

Comment:
Changes not linked with this PR

Reply:
yep it will be remove anyway in #181

Reply:
Why not fixing this point in this here as it is not related to this PR ?